common/script_values/babylon_values.txt

```diff
@@ -1574,3 +1679,56 @@ babylon_weekly_customs_union_minting_cost = {
 		subtract = modifier:country_minting_add
 	}
 }
+
+babylon_china_infamy_penalty_value = {
+	value = 0
+	if = {
+		limit = {
+			NOR = {
+				country_has_primary_culture = cu:manchu
+				country_has_primary_culture = cu:mongol
+				country_has_primary_culture = cu:tibetan
+				any_primary_culture = {
+					has_discrimination_trait = han_chinese
+				}
+			}
+		}
+		every_scope_state = {
+			limit = {
+				is_treaty_port = no
+				state_region = {
+					is_homeland = cu:han
+					NOT = {
+						this = s:STATE_FORMOSA
+					}
+				}
+			}
+			add = 5
+		}
```

Comment:
Now both the overlord and the subject get the infamy. Is this on purpose?

Reply:
Yes, that is intended.